activiti-cloud-services-org/activiti-cloud-services-org-rest/src/main/java/org/activiti/cloud/services/organization/rest/api/ModelRestApi.java

```diff
@@ -232,8 +232,8 @@ void validateModel(
 
     @ApiOperation(
             tags = MODELS,
-            value = "Validate a model content",
-            notes = "Allows to the model content without save it.")
+            value = "Validate model extensions",
+            notes = "Allows to the model extensions without save them.")
```

Comment:
Allows to "validate" is missing

Reply:
You are right, fixed here and in the validate content end point